Packages/vcs/Lib/vtk_ui/text.py

```diff
@@ -139,55 +339,64 @@ def set_font_color(self, color):
         prop.SetBackgroundColor(white_or_black(*color))
 
     def get_dimensions(self):
-        bbox = [0,0]
+        bbox = [0, 0]
         self.actor.GetSize(self.repr.GetRenderer(), bbox)
         return bbox[0], bbox[1]
 
     def show(self):
-        if self.showing() == False:
+        if self.showing() is False:
             self.place()
-            self.widget.On()
+            self.actor.VisibilityOn()
+            self.actor.Modified()
             self.render()
 
     def hide(self):
-        if self.showing() == True:
-            self.widget.Off()
+        if self.showing():
+            self.actor.VisibilityOff()
+            self.actor.Modified()
             self.render()
 
     def place(self):
```

Comment:
Had to leave a stub in, because the Textbox object subclasses this one and uses `place` to keep the cursor properly located.